app/src/main/java/org/mozilla/tv/firefox/session/SessionRepo.kt

```diff
@@ -108,4 +111,15 @@ class SessionRepo(
     }
 
     private val session: Session? get() = sessionManager.selectedSession
+
+    fun clearBrowsingData(context: Context, webViewCache: WebViewCache) {
+        sessionManager.engine.deleteData(context)
+        sessionManager.removeAll()
+        webViewCache.doNotPersist(doAfterRecreate = {
+            // Sessions must be removed prior to MainActivity being recreated, in order for initialization
+            // logic to be called. However, the URL is set before the new WebView is created, causing it to
+            // be out of sync with the Session. Removing all sessions puts them back into sync
+            sessionManager.removeAll()
```

Comment:
This was the old comment that didn't do a great job of explaining what was going on.  Suggest replacing this with the one currently in `SettingsFragment`.  (this one can be deleted)